### ISSUE TITLE
Advanced Laser Turret has a 438MJ per shot power cost

### ISSUE THREAD

**Issue by rokerboy** [veden/RampantArsenal#17]
I hope this is a bug.

This cost can overwhelm even lategame power networks, and for a turret fires upwards of once every two seconds, it's impossible to deploy in any significant number without several large nuclear reactors dedicated purely to their function. Couple this with the comparatively petty damage dealt by the shot, the end result is the turret is simply vastly too costly to use and with a horribly inefficient cost to damage ratio.

**Comment by veden** [comment]
What would you recommend as the change?

**Comment by rokerboy** [comment]
I would recommend cutting the fire rate roughly in half- maybe closer to 0.2 shots / second- and lowering the power requirement to somewhere in the 100-200MJ range.

I'm not a modder, though; my opinion comes only from playtesting.

**Comment by veden** [comment]
this should be fixed in the next version or on the master branch if you want to package the mod.

**Comment by rokerboy** [comment]
Hello.

I went through another run (with Rampant enabled as well). At 184h in, the updated turrets seem much more fairly balanced. Not so powerful as to delete entire waves on their own, but also not so pricey in terms of energy cost that they overwhelm entire power networks. I like them much more now, personally.

If it means anything, I noticed that YouTuber martincitopants also appears to use them spaced out in his own defenses now, and judging by how common they are in his walls, didn't appear to have any major issues.

https://youtu.be/uXhac1xj8DA?t=1076

Thanks, veden. :)
